### DISCUSSION TITLE
more then one pool?

### DISCUSSION THREAD

**Discussion by Jonathan4275** [Lolliedieb/lolMiner-releases#141] [Q&A]
Can you mine from more then one pool at a time useing lolminer? If so how can you do it?

**Comment by Droei** [comment]


**Reply by Jonathan4275** [reply]
Where do you do that at? In the .bat file?

**Reply by BeardedAtheist** [reply]
Yeah im lost as well

**Reply by Droei** [reply]
Ooh I probably have misunderstood the question when I read last week! Sorry about that!
